src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -960,7 +998,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     }
   }
 
-  private def isWorkflowDone: Boolean = executionStore forall isDone
+  private def isWorkflowDone: Boolean = executionStore forall { d => isDone(d) || isPreempted(d) }
```

Comment:
I had it like you say originally but I found it confusing having `isDone` return true for something else than `CallStatus.Done`. I don't really mind though.